jams/sonify.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# CREATED:2015-12-12 18:20:37 by Brian McFee <brian.mcfee@nyu.edu>
+r'''
+Sonification
+============
+
+.. autosummary::
+    :toctree: generated/
+
+    sonify
+'''
+
+import six
+import numpy as np
+import mir_eval.sonify
+from mir_eval.util import filter_kwargs
+from .eval import coerce_annotation
+from .exceptions import NamespaceError
+
+__all__ = ['sonify']
+
+
+
+def clicks(annotation, sr=22050, length=None, **kwargs):
+    '''Sonify clicks timings
+
+    '''
+
+    interval, _ = annotation.data.to_interval_values()
+
+    return filter_kwargs(mir_eval.sonify.clicks, interval[:, 0],
+                         fs=sr, length=length, **kwargs)
+
+
+def chord(annotation, sr=22050, length=None, **kwargs):
+    '''Sonify chords'''
+
+    intervals, chords = annotation.data.to_interval_values()
+
+    return filter_kwargs(mir_eval.sonify.chords,
+                         chords, intervals,
+                         fs=sr, length=length,
+                         **kwargs)
+
+
+def pitch_hz(annotation, sr=22050, length=None, **kwargs):
```

Comment:
The following code causes will cause a crash, though I think the problem is with the `mir_eval` code ([this line](https://github.com/craffel/mir_eval/blob/master/mir_eval/sonify.py#L109)), not the jams wrapper.

```python
jam = jams.load('/Users/justin/dev/jams-data/datasets/MIREX05/train05REF.jams')
ann = jam.annotations[0]
jams.sonify.pitch_hz(ann)
```
There's a division by the frequency value, but since the convention is to indicate silence (no pitch) by non-positive value, it breaks. Error:
```
---------------------------------------------------------------------------
ZeroDivisionError                         Traceback (most recent call last)
<ipython-input-9-ceb7518fe118> in <module>()
----> 1 jams.sonify.pitch_hz(ann)

/Users/justin/Documents/dev/jams/jams/sonify.pyc in pitch_hz(annotation, sr, length, **kwargs)
     61                          gram, freqs, intervals,
     62                          fs=sr, length=length,
---> 63                          **kwargs)
     64 
     65 

/Users/justin/Documents/dev/mir_eval/mir_eval/util.pyc in filter_kwargs(_function, *args, **kwargs)
    741             filtered_kwargs[kwarg] = value
    742     # Call the function with the supplied args and the filtered kwarg dict
--> 743     return _function(*args, **filtered_kwargs)
    744 
    745 

/Users/justin/Documents/dev/mir_eval/mir_eval/sonify.pyc in time_frequency(gram, frequencies, times, fs, function, length)
    127     for n, frequency in enumerate(frequencies):
    128         # Get a waveform of length samples at this frequency
--> 129         wave = _fast_synthesize(frequency)
    130         # Scale each time interval by the piano roll magnitude
    131         for m, (start, end) in enumerate((times * fs).astype(int)):

/Users/justin/Documents/dev/mir_eval/mir_eval/sonify.pyc in _fast_synthesize(frequency)
    107 
    108         # Generate ten periods at this frequency
--> 109         n_samples = int(10.0 * fs / frequency)
    110 
    111         short_signal = function(2.0 * np.pi * np.arange(n_samples) *

ZeroDivisionError: float division by zero
```